XNAControls/XNAPanel.cs

```diff
@@ -1,7 +1,18 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Rampastring.Tools;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
 using System.Globalization;
+using System.IO;
+using Color = Microsoft.Xna.Framework.Color;
+using Rectangle = Microsoft.Xna.Framework.Rectangle;
+using SixLabors.ImageSharp.Formats.Gif;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+using System.Threading;
+using System;
+using System.Collections.Generic;
```

Comment:
After the previous refactor, there's now a bunch of useless usings here.

Reply:
> After the previous refactor, there's now a bunch of useless usings here.

Only `using SixLabors.ImageSharp.Formats.Gif;` and `using SixLabors.ImageSharp.Advanced;` can be deleted, other usings are needed